### DISCUSSION TITLE
How do I get an emodji?

### DISCUSSION THREAD

**Discussion by suli9554** [burhanrashid52/PhotoEditor#544] [Q&A]
Hi. I follow the instructions from the video (https://www.youtube.com/watch?v=-XAuwfdo8tY ), where the old version 1.1.1 is used. The new version does not have the getEmoji function. Please, how can I replace this function?

![Снимок экрана 2024-01-29 143456](https://github.com/burhanrashid52/PhotoEditor/assets/72275425/ab1f8613-7db7-4c63-bc4a-220daeba63e2)
![Снимок экрана 2024-01-29 142812](https://github.com/burhanrashid52/PhotoEditor/assets/72275425/d7bfe549-69db-47f8-ba72-1cd6f2144366)

**Comment by burhanrashid52** [comment]
The `PhotoEditor.getEmoji()` method was removed in [v.1.1.4](https://github.com/burhanrashid52/PhotoEditor/blob/master/CHANGELOG.md#114) and now its part of the [sample app](https://github.com/burhanrashid52/PhotoEditor/blob/master/app/src/main/java/com/burhanrashid52/photoediting/EmojiBSFragment.kt#L91) rather than the sdk.

**Reply by CZ-224** [reply]
Hello, i have implemented this but i am unsure where the R.array.photo_editor_emoji comes from, could you tell me where in the R "array" is stored

**Reply by CZ-224** [reply]
nevermind, just found it in strings.xml thanks
